### ISSUE TITLE
Code generated from XGBoost model includes "None"

### ISSUE THREAD

**Issue by robinvanemden** [BayesWitnesses/m2cgen#183]
When transpiling `XGBRegressor` and `XGBClassifier` models such as the following basic example:

```python
from xgboost import XGBRegressor
from sklearn import datasets
import m2cgen as m2c

iris_data = datasets.load_iris(return_X_y=True)

mod = XGBRegressor(booster="gblinear", max_depth=2)
X, y = iris_data
mod.fit(X[:120], y[:120])

code = m2c.export_to_c(mod)

print(code)
```

the resulting c-code includes a Pythonesque `None` :

```c
double score(double * input) {
    return (None) + (((((-0.391196) + ((input[0]) * (-0.0196191))) + ((input[1]) * (-0.11313))) + ((input[2]) * (0.137024))) + ((input[3]) * (0.645197)));
}
```

Probably I am missing some basic step?

**Comment by StrikerRUS** [comment]
Hi @robinvanemden !
Thanks for reporting this issue! Can you please clarify versions of libraries you are using? Because I cannot reproduce the issue:
```
from xgboost import XGBRegressor
from xgboost import __version__ as xgb_version
from sklearn import datasets
import m2cgen as m2c

iris_data = datasets.load_iris(return_X_y=True)

mod = XGBRegressor(booster="gblinear", max_depth=2)
X, y = iris_data
mod.fit(X[:120], y[:120])

code = m2c.export_to_c(mod)

print("m2cgen version: {}".format(m2c.__version__))
print("XGBoost version: {}".format(xgb_version))

print(code)
```

```
[00:50:59] WARNING: C:/Jenkins/workspace/xgboost-win64_release_0.90/src/objective/regression_obj.cu:152: reg:linear is now deprecated in favor of reg:squarederror.
m2cgen version: 0.6.0
XGBoost version: 0.90
double score(double * input) {
    return (0.5) + (((((-0.314043) + ((input[0]) * (0.00603771))) + ((input[1]) * (-0.0942852))) + ((input[2]) * (0.20668))) + ((input[3]) * (0.150136)));
}
```

**Comment by robinvanemden** [comment]
Goodmorning @StrikerRUS - thanks for your quick reply! I am running the latest GitHub (head) version of the package on Ubuntu 18.04, Python 3.6.9. The requested version info: 

```
m2cgen version: 0.6.1
XGBoost version: 1.0.2
```

**Comment by robinvanemden** [comment]
Just by looking at our respective version numbers, I was able to answer my own question: by reverting to XGBoost version: 0.90, I was able to run my code example without any problem. It would be cool if it would be possible to run more recent versions of XGBoost as well, though.  Do you think that would be feasible?

**Comment by izeigerman** [comment]
Ah, it seems like in later versions the `base_score` became None by default. Which I assume should be treated as 0 perhaps? 
In any case, @robinvanemden thank you so much for reporting this. We'll definitely fix this in the following version. Meanwhile you can temporary work this around by setting the `base_score` explicitly to eg. `0.0`, like 
```XGBRegressor(base_score=0.0, ...)```

**Comment by StrikerRUS** [comment]
@robinvanemden Thank you for updates!

Indeed, after https://github.com/dmlc/xgboost/pull/5130 default `base_score` became `None` (was `0.5`). I guess it's a bug and I'll reach them out later today.

**Comment by robinvanemden** [comment]
Thanks again, @izeigerman and @StrikerRUS! Also super that you'll reach out to the xgboost devs, @StrikerRUS. It would be perfect if they'd be able to fix this in their codebase.

**Comment by StrikerRUS** [comment]
OK, they suggest using `get_params()` method to get actual values of parameters. Refer to https://github.com/dmlc/xgboost/blob/cfae247231e4977a78b255810565369c934fe1b5/tests/python/test_with_sklearn.py#L535-L537
I've created a PR with updates: #185.

**Comment by robinvanemden** [comment]
Thanks, @StrikerRUS ! I will test the PR tomorrow, is looking good.

**Comment by izeigerman** [comment]
@robinvanemden the issue has been fixed in version `0.7.0`. Please feel free to reopen this issue if you notice it again.

**Comment by robinvanemden** [comment]
@izeigerman thanks for the heads up! and will do ...
